src/docs/tezos/truffle/quickstart.md

```diff
@@ -80,7 +80,7 @@ Next, replace the box's `truffle-config.js` with the following:
 const { mnemonic, secret, password, email } = require("./faucet.json");
 
 module.exports = {
-  // see <http://truffleframework.com/docs/advanced/configuration>
+  // see <https://truffleframework.com/docs/tezos/truffle/reference/configuring-tezos-projects>
```

Comment:
`https://truffleframework.com` should technically be `https://trufflesuite.com` although we have redirect in place so ok for now

Reply:
Going to go ahead and set this to resolved (as it was meant as more of a comment than something that needs to be addressed at the moment)

Reply:
No problem. After this PR is merged I can send another one changing `truffleframework.com` to `trufflesuite.com` and verifying others links